source/localizable/smart_cart/orders_api.html.md.erb

```diff
@@ -0,0 +1,117 @@
+---
+parent: smart_cart
+title: orders_api
+---
+
+<%= partial 'partials/page_locales' %>
+
+# Orders API <%= edit_link %>
+
+Orders API allows Skroutz Merchants to perform actions on their Smart Cart orders,
+such as accepting or rejecting an order.
+
+You may find a high level overview of the service and descriptive guides for its activation in the
+[merchant support guidelines](https://<%= settings.merchants_domain %>/merchants/support/guidelines/smart_cart_integration).
+
+<%= partial 'partials/toc' %>
+
+## Setup
+
+### Obtaining an API token
+
+In order to be able to use the API, you need to generate an **API token** from within Smart Cart settings
+page in merchant's panel (Merchants > Υπηρεσίες > Έξυπνο Καλάθι).
+
+> <i class="fa fa-exclamation-triangle"></i>
+> There can only be one active **API key** per shop at a certain time.
+> Creating a new API key will expire any previous one.
+
+### Authorization
+
+Pass in a valid **API token** in the `Authorization` header, prefixed with `Bearer `
+as in the following curl example.
+
+A specific `Accept` header is also required. For more information on API requests check the [API overview](/api/v3/).
+
+<pre class="terminal">
+curl -X GET https://<%= settings.api_domain %>/merchants/ecommerce/orders/191128-1234567 \
+  -H 'Accept: application/vnd.<%= settings.site_name %>+json; version=3.0' \
+  -H 'Authorization: Bearer your_access_token_here'
+</pre>
+
+## Endpoints
+
+### Retrieve a single order
```

Comment:
We could possibly add a short sentence that prompts them to implement/use the webhook mechanism. WDYT?

Reply:
Added an intro sentence about it c19f61de